Empire - The Last Battle/Assets/Unity/Scripts/MonoBehaviours/CardSystem.cs

```diff
@@ -148,4 +149,10 @@ public class CardSystem : MonoBehaviour {
 	private void UseUpgradeCard(CardData card, Player player) {
 		throw new NotImplementedException();
 	}
+
+	private void UseTacticCard(CardData card, Player player, Unit unitToUpgrade) {
+		UnitBaseData TempUpgrade = new UnitBaseData();
```

Comment:
Missed this with the last one, but you can't create an instance of a scriptableobject with the new keyword like this.